ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/linear_optimisation/algorithms/linear_problem/LinearProblem.java

```diff
@@ -421,7 +411,7 @@ public MPConstraint getTsoRaUsedConstraint(String operator, RangeAction<?> range
     }
 
     public static double infinity() {
-        return MPSolver.infinity();
+        return 1e8;
```

Comment:
so MPSolver.infinity does not work ? 
if not, add a comment here explaining why we have to use 1e8 (an maybe increase it to 1e10 if it works as well)

Reply:
Sadly it does not, when seeing MPSolver.infinity() (which is 1e20 iirc for xpress) or above, ortools does not consider a limit at all on that side which for some reason seems to make the solver think that it's an equality constraint (?).  I will try increasing the value to 1e10